spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/testing.adoc

```diff
@@ -751,6 +751,15 @@ If you require more control over Spring REST Docs configuration than offered by
 include::{docs-java}/features/testing/springbootapplications/autoconfiguredspringrestdocs/withwebtestclient/MyRestDocsConfiguration.java[]
 ----
 
+If you want to make use of Spring REST Docs support for a parameterized output directory, you can create a `WebTestClientBuilderCustomizer` bean.
+The auto-configuration calls `entityExchangeResultConsumer` with this consumer, thereby causing each `WebTestClient` call to automatically generate the default snippets.
```

Comment:
This isn't quite right. The proposed code is correct so this description needs to be updated to match. The auto-configuration for `WebTestClient` calls the `WebTestClientBuilderCustomizer`. It's the customizer that then calls `entityExchangeResultConsumer` to set up the parameterized output.

Reply:
@wilkinsona what do you think of the following text:

> The auto-configuration calls this `WebTestClientBuilderCustomizer` in order to create the WebTestClient bean for the current context with the consumer provided in the parameter of the method `entityExchangeResultConsumer`.

Reply:
Looks good to me. Thanks.

Reply:
commit pushed d3c6997698b5e7a0f04ced8b702a0a83d0a1af1b